mne/viz/topo.py

```diff
@@ -313,6 +314,10 @@ def _imshow_tfr(ax, ch_idx, tmin, tmax, vmin, vmax, onselect, ylim=None,
         else:
             yscale = 'linear'
 
+    if yscale == "log" and is_jointplot is True and v == "2.1.0":  # noqa
+        warn("With matplotlib version 2.1.0, lines may not show up in "  # noqa
+             "`AverageTFR.plot_joint`. Upgrade to a more recent versiom.")  # noqa
```

Comment:
@larsoner I put the noqa there because otherwise it's a long-term guaranteed drop in coverage I assume ... not sure I did it right?

Reply:
I'm not sure if `# noqa` works for coverage. In any case we usually don't bother to do it.